doc/examples/applications/plot_3d_image_processing.py

```diff
@@ -0,0 +1,311 @@
+# position of this slice in the 3D dataset.
+# Note that you cannot see the Jupyter widget at work in a static HTML page,
+# as is the case in the scikit-image gallery. For the following piece of
+# code to work, you need a Jupyter kernel running either locally or in the
+# cloud: see the bottom of this page to either download the Jupyter notebook
+# and run it on your computer, or open it directly in Binder.
+
+
+def slice_in_3D(ax, i):
+    # From https://stackoverflow.com/questions/44881885/python-draw-3d-cube
+    Z = np.array([[0, 0, 0],
+                  [1, 0, 0],
+                  [1, 1, 0],
+                  [0, 1, 0],
+                  [0, 0, 1],
+                  [1, 0, 1],
+                  [1, 1, 1],
+                  [0, 1, 1]])
+
+    Z = Z * data.shape
+    r = [-1, 1]
+    X, Y = np.meshgrid(r, r)
+
+    # Plot vertices
+    ax.scatter3D(Z[:, 0], Z[:, 1], Z[:, 2])
+
+    # List sides' polygons of figure
+    verts = [[Z[0], Z[1], Z[2], Z[3]],
+             [Z[4], Z[5], Z[6], Z[7]],
+             [Z[0], Z[1], Z[5], Z[4]],
+             [Z[2], Z[3], Z[7], Z[6]],
+             [Z[1], Z[2], Z[6], Z[5]],
+             [Z[4], Z[7], Z[3], Z[0]],
+             [Z[2], Z[3], Z[7], Z[6]]]
+
+    # Plot sides
+    ax.add_collection3d(
+        Poly3DCollection(
+            verts,
+            facecolors=(0, 1, 1, 0.25),
+            linewidths=1,
+            edgecolors="darkblue"
+        )
+    )
+
+    verts = np.array([[[0, 0, 0],
+                       [0, 0, 1],
+                       [0, 1, 1],
+                       [0, 1, 0]]])
+    verts = verts * (60, 256, 256)
+    verts += [i, 0, 0]
+
+    ax.add_collection3d(
+        Poly3DCollection(
+            verts,
+            facecolors="magenta",
+            linewidths=1,
+            edgecolors="black"
+        )
+    )
+
+    ax.set_xlabel("plane")
+    ax.set_xlim(0, 100)
+    ax.set_ylabel("row")
+    ax.set_zlabel("col")
+
+    # Autoscale plot axes
+    scaling = np.array([getattr(ax,
+                                "get_{}lim".format(dim))() for dim in "xyz"])
+    ax.auto_scale_xyz(* [[np.min(scaling), np.max(scaling)]] * 3)
+
+
+def explore_slices(data, cmap="gray"):
+    from ipywidgets import interact
+    N = len(data)
+
+    @interact(plane=(0, N - 1))
+    def display_slice(plane=34):
+        fig, ax = plt.subplots(figsize=(20, 5))
+
+        ax_3D = fig.add_subplot(133, projection="3d")
+
+        show_plane(ax, data[plane], title="Plane {}".format(plane), cmap=cmap)
+        slice_in_3D(ax_3D, plane)
+
+        plt.show()
+
+    return display_slice
+
+
+explore_slices(data);
+
+#####################################################################
+# Adjust exposure
+# ===============
+# Scikit-image's `exposure` module contains a number of functions for
+# adjusting image contrast. These functions operate on pixel values.
+# Generally, image dimensionality or pixel spacing needn't be considered.
```

Comment:
```suggestion
# Generally, image dimensionality or pixel spacing doesn't need to be
# considered. In local exposure correction, though, one might want to
# adjust the window size to ensure equal size in *real* coordinates along
# each axis.
```

Reply:
Ok, I just wanted to be a little posh... :laughing: